src/features/threads/DisplayTempUb3rStatus.js

```diff
@@ -0,0 +1,27 @@
+const Feature = require("../../core/Feature");
+const Threads = require("../../sections/Threads");
+
+class DisplayTempUb3rStatus extends Feature {
+  constructor() {
+    super({
+      section: Threads,
+      name: "Display Temp Ub3r Status",
+      default: true,
+      description: "Displays the temp Ub3r status in the postbit for users who do not have permanent Ub3r.",
+      author: {
+        name: "James",
+        profile: "https://hackforums.net/member.php?action=profile&uid=2774521"
+      }
+    });
+  }
+
+  run() {
+    document.querySelectorAll("div.post_myawards > span > i[title*='Ub3r until']").forEach((award) => {
+      let uberStatus = document.createElement("span");
+      uberStatus.textContent = award.title;
+      award.parentElement.parentElement.parentElement.append(uberStatus);
```

Comment:
Would you mind styling the span that holds this text to have font size of 10? Here is an example, but you may want to test it first: 
`uberStatus.style.font-size = "10px"`

Reply:
set the fontSize to 13px instead of the 14px it defaults to. any smaller is not as easy to read, plus, 13px is the same size as the postbit stats font size.